source/adapters/opencl/kernel.cpp

```diff
@@ -358,11 +358,11 @@ UR_APIEXPORT ur_result_t UR_APICALL urKernelGetNativeHandle(
 UR_APIEXPORT ur_result_t UR_APICALL urKernelSuggestMaxCooperativeGroupCountExp(
     ur_kernel_handle_t hKernel, size_t localWorkSize,
     size_t dynamicSharedMemorySize, uint32_t *pGroupCountRet) {
-  (void)hKernel;
-  (void)localWorkSize;
-  (void)dynamicSharedMemorySize;
-  *pGroupCountRet = 1;
-  return UR_RESULT_SUCCESS;
+  std::ignore = hKernel;
```

Comment:
we usually prefer `[[maybe_unused]]` in the cl adapter, or just not naming the parameters

Reply:
Yeah, me too to be honest. I wasn't sure whether that'll be acceptable as I mostly see `std::ignore`s around the codebase. Now that you've said, I'll change it to your preference.

Reply:
Updated.